src/components/MostAffected.js

```diff
@@ -0,0 +1,10 @@
+// import React from 'react';
+// import Continent from './Continent';
+
+// const MostAffected = () => (
+//   <div>
+//     <Continent />
+//   </div>
+// );
+
+// export default MostAffected;
```

Comment:
- [ ] Kindly consider removing any commented and unused pieces of code, it makes your code base messy and difficult to navigate